### DISCUSSION TITLE
Dockerfile for Rhino App

### DISCUSSION THREAD

**Discussion by dawiegriesel** [Appsilon/rhino#357] [Q&A] [answered]
Do you have an example Dockerfile for building a Docker image for a Rhino App?

**Comment by kamilzyla** [comment, marked as answer]
Hi @dawiegriesel! Take look at our [Rhino Showcase](https://github.com/Appsilon/rhino-showcase). We are planning to update this app to the latest Rhino version and make it easy to find via our docs; the Dockerfile should work for the latest version of Rhino however.

**Reply by dawiegriesel** [reply]
Thank you for the reply @kamilzyla! The link does not seem to work, I get a 404 error
![image](https://user-images.githubusercontent.com/14890023/196988924-38b8302e-4a6c-4775-b80b-2d80aaea638c.png)

**Reply by kamilzyla** [reply]
It should work now :slightly_smiling_face:

**Reply by dawiegriesel** [reply]
Thank you it works!! Just had to copy the two yml files in. Appreciate your help!
